diesel_cli/Cargo.toml

```diff
@@ -23,6 +23,7 @@ diesel_infer_schema = "0.15.0"
 clippy = { optional = true, version = "=0.0.138" }
 
 [dev-dependencies]
+difference = "1.0"
```

Comment:
Ha, I've added the exact same thing locally to test something but didn't refactor anything so I didn't push it!